### PR TITLE
Adds logging for organic modal experiment

### PR DESCRIPTION
**Pull request by holytoastr**
## Summary

- StatsD logging added to AcceptableVerifiedCredentialAdoptionService to track number of conversion attempts
- Login Adoption Team

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#56089
- [Backend Logging - 56089](https://app.zenhub.com/workspaces/logingov-adoption-team-632280b31e745000136a61fc/issues/gh/department-of-veterans-affairs/va.gov-team/56089)

## Testing done

- Automated testing to confirm stats are logged
- Manual testing can be done by checking statsd log files after logging in with a dslogon user without an avc/ivc

## Screenshots
NA

## What areas of the site does it impact?
- Adds additional statsd logging

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [ ]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [ ]  Feature/bug has a monitor built into Datadog or Grafana (if applicable)
- [x]  If app impacted requires authentication, did you login to a local build and verify all authenticated routes work as expected
- [ ]  I added a screenshot of the developed feature

## Requested Feedback

(OPTIONAL)_What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

File: app/services/acceptable_verified_credential_adoption_service.rb [hunk 1]
```diff
@@ -49,4 +50,15 @@ def verified_credential_at?
...
+  def log_results(conversion_type)
+    StatsD.increment("#{stats_key}.conversion_attempt", tags: [
+                       "conversationType:#{conversion_type}",
+                       "credentialType:#{credential_type}"
+                     ])
+  end
+
+  def stats_key
+    'api.user_transition_availability'
+  end
```
Comment (rileyanderson) [inline app/services/acceptable_verified_credential_adoption_service.rb:63]: I would suggest putting the `conversion_type` and `credential_type` as namespaces in your key and not worrying about tags. 
Something like: `api.user_transition_availability.organic_modal.ds_logon` etc. 

Tags are used more for metadata fields such as `env`, `host`, etc (which are auto populated).

Comment (holytoastr) [reply app/services/acceptable_verified_credential_adoption_service.rb:63]: Good to know! Thank you! I will make that change!

Comment (holytoastr) [reply app/services/acceptable_verified_credential_adoption_service.rb]: @rileyanderson Would you mind giving it a second review when you get a chance? Thanks! 🙇‍♀️

**Review by ryan-mcneil — APPROVED** [review]
LGTM!

**Pull request merged**